mycodo/mycodo_flask/utils/utils_trigger.py

```diff
@@ -172,13 +172,13 @@ def trigger_activate(trigger_id):
     actions = Actions.query.filter(
         Actions.function_id == trigger_id)
 
-    if not actions.count():
+    if not actions.count() and mod_trigger.trigger_type != 'trigger_run_pwm_method':
         error.append("No Actions found: Add at least one Action before activating.")
 
     for each_action in actions.all():
         error = check_actions(each_action, error)
 
-    if mod_trigger.trigger_type == 'run_pwm_method':
+    if mod_trigger.trigger_type == 'trigger_run_pwm_method':
```

Comment:
Debug output told me, this would include the "trigger_" prefix. **CAUTION: More places in this function skip the prefix, too. Might they suffer the same issue?**